### ISSUE TITLE
It's REALLY hard to visualize LLM sampling, but you made it easier!

### ISSUE THREAD

**Issue by Hellisotherpeople** [Artefact2/llm-sampling#3]
Writing this from ICLR 2025. You helped us get an oral presentation for our paper on min_p, and we used screenshots of this demo in our poster session. Thank you. Would love to work with you on visualization tech for LLM logits. 

https://arxiv.org/abs/2407.01082

**Comment by shmulc8** [comment]
I'm currently writing a post about LLM sampling, a topic that inspired me after attending a lecture on Min-p.
I made some updates to the code to make it usable again (https://github.com/anuk909/llm-sampling), and I’d love to collaborate on developing more visualizations if you’re interested.
